test/type_5_test.c

```diff
@@ -177,6 +177,71 @@ static void test_streamed_streamed_kv_map(void **state) {
   assert_null(map);
 }
 
+static void test_get(void **state) {
```

Comment:
Please refactor this to be structured as one set of localized assertions per testcase

Reply:
done